### DISCUSSION TITLE
App is blank

### DISCUSSION THREAD

**Discussion by Mrdev88** [hydraulic-software/conveyor#132] [Q&A]
I made a electron app with conveyor but it's blank and I'm receiving a lot of errors in my console.
<img width="1229" alt="Screenshot 2024-11-05 at 7 07 21 PM" src="https://github.com/user-attachments/assets/e92cde2c-207a-4fc0-bf33-ed10c232600f">

<img width="1353" alt="Screenshot 2024-11-05 at 7 08 11 PM" src="https://github.com/user-attachments/assets/366eb9d5-e64a-482b-8149-15a6b730a738">
All the files are there but it says file not found

**Comment by mikehearn** [comment]
Can you show us your config file?
